eng/common/pipelines/ci.yml

```diff
@@ -0,0 +1,107 @@
+trigger: none
+pr:
```

Comment:
I assume this should be updated?

For typespec core I think also we would like to have this run on `main`